Dockerfile

```diff
@@ -10,7 +10,11 @@ WORKDIR /app
 RUN apt-get update && apt-get install -y \
   git \
   netcat \
-  && rm -rf /var/lib/apt/lists/*
+  libcairo2-dev \
+  libpango1.0-dev \
+  libjpeg-dev \
+  libgif-dev \
+  librsvg2-dev   && rm -rf /var/lib/apt/lists/*
```

Comment:
Why are these changes introduced?

Reply:
they should be indeed. They were removed begin of may (not sure why, but dont seem part of this story)